pkg/apis/operator/v1alpha1/glusternode_types.go

```diff
@@ -0,0 +1,45 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
Do we need this here ?

Reply:
@shtripat will remove this when he posts his PR w/ the types.